pyproject.toml

```diff
@@ -67,7 +67,6 @@ known_third_party = [
 target-version = ['py36', 'py38']
 
 [tool.coverage.report]
-fail_under = 95
```

Comment:
I think this change should have been made a while ago. We want to enforce combined coverage of the entire test suite, not of individual test workflows.